include/deal.II/hp/fe_collection.h

```diff
@@ -260,6 +268,18 @@ namespace hp
     unsigned int
     max_dofs_per_cell() const;
 
+    /**
+     * @}
+     */
+
+    /**
+     * @name Functions to support hp adaptivity
+     * @{
+     */
+
+    /**
+     * @{
+     */
```

Comment:
```suggestion
```
Duplicated opening bracket.

Reply:
I fixed that by removing the other opening brace.